tests/test_toolargparse.py

```diff
@@ -101,6 +101,34 @@
 outputs: []
 """
 
+script_f = """
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: ExpressionTool
+
+inputs:
+  foo: int[]
```

Comment:
Please also add a `long` test with numbers bigger than `2^31` and smaller than `-1*(2^31)`